app/src/main/kotlin/com/intive/picover/parties/viewmodel/PartyDetailsViewModel.kt

```diff
@@ -4,25 +4,39 @@ import androidx.compose.runtime.MutableState
 import androidx.compose.runtime.mutableStateOf
 import androidx.lifecycle.SavedStateHandle
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.intive.picover.parties.model.toUI
 import com.intive.picover.parties.repository.MockedPartiesRepository
 import com.intive.picover.parties.state.PartyDetailsState
+import com.intive.picover.parties.state.PartyDetailsState.Error
+import com.intive.picover.parties.state.PartyDetailsState.Loaded
+import com.intive.picover.parties.state.PartyDetailsState.Loading
 import dagger.hilt.android.lifecycle.HiltViewModel
 import javax.inject.Inject
+import kotlinx.coroutines.launch
 
 @HiltViewModel
 class PartyDetailsViewModel @Inject constructor(
 	savedStateHandle: SavedStateHandle,
 	private val partiesRepository: MockedPartiesRepository,
 ) : ViewModel() {
 
-	val partyDetails: MutableState<PartyDetailsState> = mutableStateOf(PartyDetailsState.Loading)
+	val partyDetails: MutableState<PartyDetailsState> = mutableStateOf(Loading)
 	private val partyId: Int = savedStateHandle["partyId"]!!
 
 	init {
-		loadParty(partyId)
+		viewModelScope.launch {
+			loadParty(partyId)
```

Comment:
same like here: PartiesViewModel.kt

Reply:
for sure not, especially when we want to use a retry mechanism soon :D i wanted to fix it in separate task for error/retry mechanism impl but i will fix it now :)